json_annotation/lib/src/checked_helpers.dart

```diff
@@ -125,7 +135,9 @@ class CheckedFromJsonException implements Exception {
         'CheckedFromJsonException',
         if (_className != null) 'Could not create `$_className`.',
         if (key != null) 'There is a problem with "$key".',
-        if (message != null) message,
-        if (message == null && innerError != null) innerError.toString(),
+        if (message != null)
+          message!
```

Comment:
Use `{}` now that this got broken onto its own line

Reply:
Cannot use `{}` within list comprehension syntax. :shrug: